parser_test.go

```diff
@@ -332,6 +332,147 @@ const expectedListOutput = `
 }
 `
 
+const expectedInventoryOutput = `[all]
+10.0.0.1
+10.0.0.10
+10.0.0.7
+10.0.0.8
+10.0.0.9
+10.0.1.1
+10.120.0.226
+10.2.1.5
+10.20.30.40
+192.168.0.3
+50.0.0.1
+
+[all:vars]
+datacenter=null
+ids=null
+map=null
+olddatacenter=null
```

Comment:
Looks like the above error manifests here. These should probably match the vars at :284, in expectedListOutput.

Reply:
Wow. I'm sorry.

Reply:
It's no problem, I almost missed it myself.